packages/common/src/index.ts

```diff
@@ -796,6 +797,7 @@ export type CreateDatabricksCredentials = {
     type: WarehouseTypes.DATABRICKS;
     serverHostName: string;
     port: number;
+    dbname: string;
```

Comment:
I think we should rename to catalog as it seems consistent with the dbt profile naming
https://docs.getdbt.com/reference/warehouse-setups/databricks-setup

And we can also make it optional as we default to SPARK. What you think?

Reply:
agreed on rename. we should not default to SPARK anymore. it does not make any sense. I am not sure how ODBC connection was handling that but default catalog is `hive_metastore` << this is also true for our staging env.

Reply:
Is the `initialCatalog` arg optional ? if so, I'm ok with not have SPARK as a fallback.

Reply:
it is, already removed spark as it does not make any sense after switching the databricks client